Source/WebCore/PAL/pal/crypto/commoncrypto/CryptoDigestCommonCrypto.mm

```diff
@@ -142,6 +142,11 @@
     }
 }
 
+void CryptoDigest::addBytes(std::span<const uint8_t> input)
+{
+    return addBytes(reinterpret_cast<const void*>(input.data()), input.size());
```

Comment:
I doubt we need any casting here. (If we really need a cast, then static_cast would likely suffice)